### ISSUE TITLE
bug: C/W ratio silently passes when cached_max_loss is null — false compliance

### ISSUE THREAD

**Issue by leviceroy** [leviceroy/theta-vault#62]
## Bug — Tom Sornoff Playbook Audit

**File:** `src/calculations/compliance.ts` lines 176–191

**Problem:** The C/W ratio check silently passes when `cached_max_loss` is null — no violation, no flag:

```typescript
const maxLossPerContract = trade.cached_max_loss != null
  ? trade.cached_max_loss / (trade.quantity ?? 1) / 100
  : null;
if (maxLossPerContract != null && maxLossPerContract > 0) {
  // check runs — but if maxLossPerContract is null, block is skipped entirely
}
```

Imported trades frequently have `cached_max_loss = null`. A trade with a 15% C/W ratio on a playbook requiring ≥33% passes as fully compliant.

**Fix:** When `cached_max_loss` is null but `spread_width` and `credit_received` are available, derive `maxLossPerContract = spread_width - credit_received`. Or push a "C/W Ratio — unable to verify (max loss not available)" informational violation so the trader knows the check was skipped.
